GG/src/Font.cpp

```diff
@@ -240,6 +240,21 @@ namespace {
     };
     const boost::xpressive::function<PushSubmatchOntoStack>::type Push = {{}};
 
+    struct PushSubmatchOntoStackP
+    {
+        typedef void result_type;
+        void operator()(const std::string* str,
+                        std::stack<Font::Substring>& tag_stack,
+                        bool& ignore_tags,
+                        const boost::xpressive::ssub_match& sub) const
+        {
+            tag_stack.push(Font::Substring(*str, sub));
+            if (tag_stack.top() == PRE_TAG)
+                ignore_tags = true;
```

Comment:
That is correct. This is the existing behavior.  It is also documented in the comments of Font.h